source/Integration_overview.rst

```diff
@@ -124,46 +124,49 @@ Requests from the client browser require a configured :ref:`CORS policy<cors-pol
+
+The table below describes six pairs of environment connections between TradingView and the broker.
+
++-------------------------+--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
+| TradingView environment | Broker environment | Description                                                                                                                                         |
++=========================+====================+=====================================================================================================================================================+
+| Localhost               | Staging            | An environment that TradingView developers use for troubleshooting.                                                                                 |
++-------------------------+--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
+| Staging                 | Staging            | An environment that the broker uses during integration development in the TradingView sandbox.                                                      |
++-------------------------+--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
+| Production              | Staging            | An environment that TradingView and the broker use to test any changes the broker made after release to the TradingView production environment.     |
+|                         |                    | Used to ensure that changes do not affect the broker production environment.                                                                        |
++-------------------------+--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
+| Localhost               | Production         | An environment that TradingView developers use for troubleshooting.                                                                                 |
++-------------------------+--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
+| Staging                 | Production         | A pre-release environment that TradingView developers use for testing features implemented by the broker.                                           |
+|                         |                    | During this stage, TradingView performs the final testing of the broker's production API and the deployment of the TradingView client applications. |
```

Comment:
As an idea to simplify the sentence: `perform testing` and `perform deployment` might be replaced with the verbs. 
```suggestion
|                         |                    | During this stage, TradingView tests the broker's production API for the last time and deploys the TradingView client applications. |
```